settings.gradle.kts

```diff
@@ -372,3 +372,7 @@ include("sdks:java:io:iceberg:bqms")
 findProject(":sdks:java:io:iceberg:bqms")?.name = "bqms"
 include("it:clickhouse")
 findProject(":it:clickhouse")?.name = "clickhouse"
+include("sdks:java:extensions:sql:iceberg")
+findProject(":sdks:java:extensions:sql:iceberg")?.name = "iceberg"
+include("examples:java:iceberg")
+findProject(":examples:java:iceberg")?.name = "iceberg"
```

Comment:
We also need to add `sql/iceberg` module to sqlprecommit and sqlpostcommit:

https://github.com/apache/beam/blob/1de583037dc57bde439104b74bea4b3a5ada0d04/build.gradle.kts#L377

https://github.com/apache/beam/blob/1de583037dc57bde439104b74bea4b3a5ada0d04/build.gradle.kts#L425

and also add examples/iceberg module to java precommit, and if any examples also get exercised in postcommit test, we need to add it there as well

Reply:
> We currently don't have any tests for examples

The `build` task will still compile the module regardless there is test or not. If not add `:examples:iceberg` to JavaPreCommit we won't notice when there is compile error introduced.

Reply:
Done

Reply:
Thanks, just added them